services/vehicle/src/main/java/com/wedriveu/services/vehicle/entity/Vehicle.java

```diff
@@ -14,6 +14,10 @@
  */
 public class Vehicle {
 
+    public static final String STATUS_STARTED = "started";
```

Comment:
You misunderstood, i meant to put here all the `Vehicle` statuses, not the `Booking` ones